Source/SwiftLintFramework/Rules/Lint/UnusedImportRule.swift

```diff
@@ -225,13 +234,15 @@ private extension SwiftLintFile {
 
     func rangedAndSortedUnusedImports(of unusedImports: [String], contents: NSString) -> [(String, NSRange)] {
         return unusedImports
-            .map { module in
-                let testableImportRange = contents.range(of: "@testable import \(module)\n")
-                if testableImportRange.location != NSNotFound {
-                    return (module, testableImportRange)
+            .compactMap { module in
+                // We can't use raw string literals because it breaks SourceKit
+                // https://bugs.swift.org/browse/SR-11099
+                // When we require Swift 5.2 or later to build SwiftLint we can switch this back.
+                // let pattern = #"^(@.+\s+)?import\s+\#(module)\b.*?\n"#
+                let pattern = "^(@.+\\s+)?import\\s+\(module)\\b.*?\\n"
```

Comment:
In surprised that `\n` needs to escaping

Reply:
You're right. Unescaped in efc75042.